app/src/pages/inside/common/modals/editItemModal/editItemModal.jsx

```diff
@@ -105,6 +105,10 @@ const messages = defineMessages({
     id: 'TestItemDetailsModal.parametersLabel',
     defaultMessage: 'Parameters:',
   },
+  descriptionHint: {
+    id: 'EditItemModal.descriptionAdviceHint',
+    defaultMessage: 'You used 1000 of 2048 symbols',
```

Comment:
This is static text, so the user can enter 2048 characters but still see that text, maybe the idea was to dynamically count?

Reply:
I'll ask UX.

Reply:
UX said that thought about static text, but if possible we can improve it to dynamic